stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/annotation.bal

```diff
@@ -37,6 +40,9 @@ public annotation configuration attach service<> {
     string sslEnabledProtocols;
     string ciphers;
     string sslProtocol;
+    boolean certificateRevocationVerifier;
+    int cacheSize;
+    int cacheDelay;
```

Comment:
Is this a cache specific to certificate handling? If so, can we rename these fields to something which reflects that?